### PR TITLE
GUI-2193, GUI-2198

### PR DESCRIPTION
**Pull request by coyote240**
https://eucalyptus.atlassian.net/browse/GUI-2193
Create alarm from alarms landing page.  In the wireframes, we now have the dialog direct the user to the metrics landing page.

https://eucalyptus.atlassian.net/browse/GUI-2198
Implements both single and multi-select deletion of alarms.

### CODE REVIEW

File: eucaconsole/templates/cloudwatch/alarms.pt [hunk 1]
```diff
@@ -116,6 +116,30 @@
...
+            <div id="create-alarm-modal" class="reveal-modal small" data-reveal="">
+                <a class="close-reveal-modal" href="#">&#215;</a>
+                <div class="row">
+                    <p i18n:translate="">A metric must be selected before you can create an alarm.</p>
+                    <p i18n:translate="">Choose the action "Create Alarm" when viewing a metric from
+                    the monitoring tab on any resource's detail page, or from the
+                    metrics landing page.</p>
+                </div>
+                <div class="row">
+                    <a class="button expand" href="${request.route_path('cloudwatch_metrics')}">
```
Comment (kamalgill) [inline eucaconsole/templates/cloudwatch/alarms.pt]: @coyote240 Need an i18n:translate attribute here

File: eucaconsole/templates/dialogs/delete_alarm_dialog.pt [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+            <h6 class="title">
+                <a ng-click="toggleContent()">
+                    <i class="fi-plus" ng-show="!expanded"></i>
+                    <i class="fi-minus" ng-show="expanded"></i>
+                    &nbsp;<span i18n:translate="">{{ alarms.length }} alarms</span>
+                </a>
+            </h6>
+            <div class="content" ng-show="expanded">
+                <div class="row header">
+                    <div class="large-4 columns">
```
Comment (kamalgill) [inline eucaconsole/templates/dialogs/delete_alarm_dialog.pt]: @coyote240 Need an i18n:translate attribute here

File: eucaconsole/templates/dialogs/delete_alarm_dialog.pt [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+                        Alarm Name
+                    </div>
+                    <div class="large-8 columns">
```
Comment (kamalgill) [inline eucaconsole/templates/dialogs/delete_alarm_dialog.pt]: @coyote240 Need an i18n:translate attribute here

File: eucaconsole/forms/alarms.py [hunk 4]
```diff
@@ -193,12 +193,5 @@ def get_unit_choices():
         return choices
 
 
-class CloudWatchAlarmDeleteForm(BaseSecureForm):
```
Comment (kamalgill) [inline eucaconsole/forms/alarms.py]: @coyote240: We'll need to reinstate this to provide a CSRF-protected form for the alarms delete operation.

File: eucaconsole/views/alarms.py [hunk 5]
```diff
@@ -127,27 +129,37 @@ def cloudwatch_alarms_create(self):
             self.request.error_messages = error_msg_list
         return self.render_dict
 
-    @view_config(route_name='cloudwatch_alarms_delete', renderer=TEMPLATE, request_method='POST')
+    @view_config(route_name='cloudwatch_alarms_delete', renderer='json', request_method='DELETE')
     def cloudwatch_alarms_delete(self):
-        if self.delete_form.validate():
```
Comment (kamalgill) [inline eucaconsole/views/alarms.py]: @coyote240: Just a heads up that delete_form.validate() runs the CSRF check, so removing it here leaves alarm deletion vulnerable to CSRF attack vectors.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/1428/
Test FAILed.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/1430/
Test PASSed.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/1431/
Test FAILed.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com//job/user-console-PR-monitor/1432/
Test PASSed.

**Pull request merged**